### PR TITLE
Create es_es.lang file

### PR DESCRIPTION
**Pull request by ghost**
This strings can be used too for any dialect of Spanish.

### CODE REVIEW

File: src/main/resources/assets/betterfps/lang/es_es.lang [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+betterfps.installer.button.extract=Extraer
+betterfps.installer.button.algorithm=Probar Algoritmos
+betterfps.installer.button.downloads=Descargas
+betterfps.installer.button.issues=Errores/Sugerencias
+betterfps.installer.label.game-dir=Directorio de instalación de Minecraft
+betterfps.installer.label.version=Versión
+betterfps.installer.label.profile=Perfile
+betterfps.installer.done=¡Listo! (en %sms)
+betterfps.installer.algorithm.title=Comprobar Algoritmo
+betterfps.installer.algorithm.button.change=Cambiar Slgoritmo a %s
```
Comment (Guichaguri) [inline src/main/resources/assets/betterfps/lang/es_es.lang]: I guess you mistyped "Algoritmo" here

File: src/main/resources/assets/betterfps/lang/es_es.lang [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+betterfps.installer.algorithm.button.again=Comprobar otra vez
+betterfps.installer.algorithm.working=Comprobando Slgoritmos...
```
Comment (Guichaguri) [inline src/main/resources/assets/betterfps/lang/es_es.lang]: Same here

**Review by Guichaguri — CHANGES_REQUESTED** [review]

**Comment by Guichaguri** [comment]
Thanks! Can you check what I pointed out?

**Comment by ghost** [comment]
Done, thanks for pointing it out!

**Review by Guichaguri — APPROVED** [review]

**Comment by Guichaguri** [comment]
Thank you! :D

**Comment by ghost** [comment]
No prob! Thank u for making this mod!

**Pull request merged**